Makefile

```diff
@@ -272,18 +264,38 @@ rhel84-nvidia: ## Build RHEL 8.4 image with GPU support
 	$(if $(ADDITIONAL_OVERRIDES),--overrides=${ADDITIONAL_OVERRIDES}) \
 	--aws-instance-type p2.xlarge
 
-.PHONY: rhel79
-rhel79: build
-rhel79: ## Build RHEL 7.9 image
-	./bin/konvoy-image build images/ami/rhel-79.yaml \
+.PHONY: rhel84-ova
+rhel84-ova: build
+rhel84-ova: ## Build RHEL 8.4 image
+	./bin/konvoy-image build images/ova/rhel-84.yaml \
 	--dry-run=$(BUILD_DRY_RUN) \
 	-v ${VERBOSITY} \
 	$(if $(ADDITIONAL_OVERRIDES),--overrides=${ADDITIONAL_OVERRIDES})
 
-.PHONY: rhel79-ova
-rhel79-ova: build
-rhel79-ova: ## Build RHEL 7.9 image
-	./bin/konvoy-image build images/ova/rhel-79.yaml \
+.PHONY: rhel84-ova-offline
```

Comment:
Makefile target clutter is getting worse. I will have a PR in next sprint to fix this.